src/hookable.ts

```diff
@@ -118,6 +119,13 @@ export class Hookable <
     }
     return serial(this._hooks[name], fn => fn(...args as any))
   }
+
+  callHookSync<NameT extends HookNameT> (name: NameT, ...args: Parameters<InferCallback<HooksT, NameT>>) {
+    if (!this._hooks[name]) {
+      return
+    }
+    return this._hooks[name].map(fn => fn(...args as any))
```

Comment:
We need to ensure that hooks are not async otherwise it breaks reliability for hook consumers when their promises will be simply discarded as if we call their hooks with this method. This generally looks like an anti-pattern to me to introduce in bookable for one specific use case of nuxt

Reply:
Happy to close this if you think so. I think any promises will be returned as an array though, to be handled by the user.

Reply:
What if we make `serial` implementation configurable for `callHook` with something like like `callHookWith(callFn, name, ...args`)? This way we can have a way to do custom checks like accessing raw returned values to validate but not introducing `sync` to the core